honeybee_vtk/helper.py

```diff
@@ -0,0 +1,147 @@
+
+    # Create a Ladybug Face3D object from boundary points and hole points
+    if holes_points:
+        # Create Ladybug Point3D objects from hole points
+        lb_holes_pts = [
+            [Point3D(point[0], point[1], point[2]) for point in points]
+            for points in holes_points]
+
+        face = Face3D(boundary=lb_boundary_pts, holes=lb_holes_pts)
+
+    else:
+        face = Face3D(boundary=lb_boundary_pts)
+
+    # Generate triangles from the Face3D object
+    triangles = face.triangulated_mesh3d
+
+    # A list of lists with each list having vertices of triangles
+    points_lst = [[
+        triangles.vertices[face[0]], triangles.vertices[face[1]],
+        triangles.vertices[face[2]]] for face in triangles.faces]
+
+    return points_lst
+
+
+def check_convex(boundary_points):
+    """Check whether a Face3D is convex or not.
+
+    This function creates a Ladybug Face3D object from boundary points and reports
+    whether the Face3D is convex or not.
+
+    Args:
+        boundary_points: A list of list of points. Here, each point is a list of 
+            X, Y, and Z cordinates of the point.
+
+    Returns:
+        A boolean value.
+    """
+
+    lb_boundary_pts = [
+        Point3D(point[0], point[1], point[2]) for point in boundary_points]
+
+    face = Face3D(boundary=lb_boundary_pts)
+
+    return face.is_convex
+
+
+def get_point3d(points):
+    """Convert a list of points in to a list of Ladybug Point3D objects.
+
+    Args:
+        points: A list of points. Here, each point is a list of 
+            X, Y, and Z cordinates of the point.
+
+    Returns:
+        A list of Ladybug Point3D objects.
+    """
+    return [Point3D(point[0], point[1], point[2]) for point in points]
+
+
+def get_end_point(point, vector):
+    """Move a point in the direction of a vector and return the moved point.
+
+    Here, each point is a list of 
+            X, Y, and Z cordinates of the point.
+
+    Args:
+        point: A point as a list where the list has X, Y, and Z cordinates of the point.
+        vector: A vector as a list where the list has X, Y, and Z component of the
+            vectors.
+
+    Returns:
+        A point as a list where the list has X, Y, and Z cordinates of the point.
+    """
+    return [point[0] + vector[0], point[1] + vector[1], point[2] + vector[2]]
+
+
+def face_center(points):
+    """Get center point and normal for a Ladybug Face3D.
+
+    This function creates a Ladybug Face3D from the points and returns its center and
+    normal properties.
+
+    Args:
+        points: A list of Ladybug Point3D objects that can be a boundary for a face.
+
+    Returns:
+        A tuple with two elements
+
+        - face.center: The center point of the face as a Ladybug Point3D object.
+
+        - face.normal: The face normal for the Ladybug Face3D.
+    """
+    face = Face3D(boundary=points)
+    return face.center, face.normal
+
+
+def get_face_center(points_lst):
```

Comment:
I have a few comments about this function.

1. names like `points_lst` makes me cringe. let's use annotations. points: List[List[Tuple]]]
2. it looks like you want to change `face_center` function to `_face_center` and then rename this function to `face_center`. Right now having `face_center` and `get_face_center` can be really confusing.
3. finally, this function doesn't return face_center as far as I can see!